support nest 7

Is it possible to support nest 7?
using npm I get this error
```
npm ERR! While resolving: example-server@1.0.0
npm ERR! Found: @nestjs/common@7.6.5
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^7.6.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^6.4" from nest-next@9.3.0
npm ERR! node_modules/nest-next
npm ERR!   nest-next@"^9.3.0" from the root project
```

--- a/examples/basic/package.json
+++ b/examples/basic/package.json
@@ -12,9 +12,9 @@
     "start": "cross-env NODE_ENV=production node .next/production-server/src/main.js"
   },
   "dependencies": {
-    "@nestjs/common": "^6.5.3",
-    "@nestjs/core": "^6.5.3",
-    "@nestjs/platform-express": "^6.5.3",
+    "@nestjs/common": "^7.6.5",
+    "@nestjs/core": "^7.6.5",
+    "@nestjs/platform-express": "^7.6.5",
     "cross-env": "^6.0.3",
     "nest-next": "^9.3.0",
     "next": "10",

--- a/examples/monorepo/server/package.json
+++ b/examples/monorepo/server/package.json
@@ -6,9 +6,9 @@
   "types": "src/main.ts",
   "license": "MIT",
   "dependencies": {
-    "@nestjs/common": "^6.10.12",
-    "@nestjs/core": "^6.10.12",
-    "@nestjs/platform-fastify": "^6.10.12",
+    "@nestjs/common": "^7.6.5",
+    "@nestjs/core": "^7.6.5",
+    "@nestjs/platform-fastify": "^7.6.5",
     "cross-env": "^6.0.3",
     "nest-next": "^9.1.0",
     "next": "^9.1.6",

--- a/package.json
+++ b/package.json
@@ -19,9 +19,9 @@
     "lint": "node ./node_modules/eslint/bin/eslint.js lib/**"
   },
   "devDependencies": {
-    "@nestjs/common": "^6.4",
-    "@nestjs/core": "^6.4",
-    "@nestjs/platform-fastify": "^6.4",
+    "@nestjs/common": "^7.6.5",
+    "@nestjs/core": "^7.6.5",
+    "@nestjs/platform-fastify": "^7.6.5",
     "@types/node": "^11.12",
     "@typescript-eslint/eslint-plugin": "^2.18.0",
     "@typescript-eslint/parser": "^2.18.0",
@@ -42,8 +42,8 @@
     "rxjs": "^6.4"
   },
   "peerDependencies": {
-    "@nestjs/common": "^6.4",
-    "@nestjs/core": "^6.4",
+    "@nestjs/common": ">=6.4",
+    "@nestjs/core": ">=6.4",
     "next": ">=9"
   }
 }

